NexusEmbeddedEditor/NexusEmbeddedEditor/Window/RobloxStudioWindow.cs

```diff
@@ -16,18 +16,72 @@
 
 namespace NexusEmbeddedEditor.Window
 {
+
+    /*
+     * An AutomationElement wrapper meant for debugging the Automation hierarchy.
+     * This should NOT be used in production.
+     * 
+     * Usage:
+        var debug = new DebugElement(this.Window.Window);
+        
+        Console.WriteLine("Debugging...");  // set breakpoint here
+
+     * There should now be a tree in the Locals widget containing every element.
+     */
+    /*
+    public class DebugElement
```

Comment:
I am not a fan of having this code commented out with the intent that developers will un-comment it out for testing. Undoing it may be overlooked. Can using this be a command-line argument option?

Reply:
Would excluding it with an `#if` directive be alright?

```c#
namespace NexusEmbeddedEditor.Window
{
#if DEBUG
    /*
     * doc
     */
    public class DebugElement
    {
        // code
    }
#endif

    // ...
```

This removes it from the build if the `Debug` configuration in the top ribbon bar is changed to `Release`, causing compile errors so you don't accidentally reference `DebugElement` in production.

Reference: https://stackoverflow.com/questions/337431/removing-code-from-release-build-in-net

Reply:
That would be better. Expanding the `#if` usage for the actual wrapping call would be better since it would reduce the risk of accidentally committing debug code. I say that as someone who hasn't used the debug code though.